tests/TestCase.php

```diff
@@ -288,4 +288,15 @@ protected function placeholderQuery(): string
         }
         ';
     }
+
+    /**
+     * Get the test resolver.
+     *
+     * @param  string  $method
+     * @return string
+     */
+    protected function getResolver(string $method = 'resolve'): string
```

Comment:
`get` indicicates this returns a property, so maybe we can rename that, ideas:
- qualifyResolverMethod
- qualifiedTestResolver

Open to suggestions.

Reply:
👍